subprojects/gradle/cd/build.gradle.kts

```diff
@@ -36,6 +36,13 @@ dependencies {
     testImplementation(project(":gradle:test-project"))
 }
 
+configurations.all {
+    resolutionStrategy {
+        // docker client тянет 4.x версию
+        force(Dependencies.okhttp)
```

Comment:
i think it is not necessary even without bom

Reply:
Removed